packages/data-transformation/fhir-to-csv/src/parseFhir/configurations/config_Condition.ini

```diff
@@ -10,79 +10,98 @@ meta_versionid = meta.versionId
 meta_lastupdated = meta.lastUpdated
 meta_source = meta.source
 identifier_0_system = identifier.0.system
+identifier_0_type_text = identifier.0.type.text
 identifier_0_value = identifier.0.value
-clinicalstatus_coding_0_system = clinicalStatus.coding.0.system
-clinicalstatus_coding_0_code = clinicalStatus.coding.0.code
-category_0_coding_0_system = category.0.coding.0.system
+identifier_1_system = identifier.1.system
+identifier_1_type_text = identifier.1.type.text
+identifier_1_value = identifier.1.value
+identifier_2_system = identifier.2.system
+identifier_2_type_text = identifier.2.type.text
+identifier_2_value = identifier.2.value
+identifier_3_system = identifier.3.system
+identifier_3_type_text = identifier.3.type.text
+identifier_3_value = identifier.3.value
+identifier_4_system = identifier.4.system
+identifier_4_type_text = identifier.4.type.text
+identifier_4_value = identifier.4.value
+subject_reference = subject.reference
+# sort alphabetically from here on
+abatementage_code = abatementAge.code
+abatementage_system = abatementAge.system
+abatementage_unit = abatementAge.unit
+abatementage_value = abatementAge.value
+abatementdatetime = abatementDateTime
+abatementperiod_end = abatementPeriod.end
+abatementperiod_start = abatementPeriod.start
+abatementrange_high_code = abatementRange.high.code
+abatementrange_high_system = abatementRange.high.system
+abatementrange_high_unit = abatementRange.high.unit
+abatementrange_high_value = abatementRange.high.value
+abatementrange_low_code = abatementRange.low.code
+abatementrange_low_system = abatementRange.low.system
+abatementrange_low_unit = abatementRange.low.unit
+abatementrange_low_value = abatementRange.low.value
+abatementstring = abatementString
```

Comment:
It was missing, re: https://linear.app/metriport/issue/ENG-857/analytics-platform-data-improvements